app/controllers/submission_comments_api_controller.rb

```diff
@@ -18,12 +18,84 @@
 
 # @API Submission Comments
 #
-# This API can be used to create files to attach to submission comments.  The
+# This API can be used to edit and delete submission comments.
+# It can also be used to create files to attach to submission comments. The
 # submission comments themselves can be created using the Submissions API.
 class SubmissionCommentsApiController < ApplicationController
   before_action :require_context
 
   include Api::V1::Attachment
+  include Api::V1::SubmissionComment
+
+  # @API Edit a submission comment
+  #
+  # Edit the given submission comment.
+  #
+  # @argument comment [String]
+  #   If this argument is present, edit thet text of a comment.
+  #
+  # @argument attachment_ids[] [Integer]
+  #   Attach files to this comment that were previously uploaded using the
+  #   Submission Comment API's files action.
+  #
+  # @argument media_comment_id [String]
+  #   Add an audio/video comment to the submission. Media comments can be added
+  #   via this API, however, note that there is not yet an API to generate or
+  #   list existing media comments, so this functionality is currently of
+  #   limited use.
+  #
+  # @argument media_comment_type [String, "audio"|"video"]
+  #   The type of media comment being added.
```

Comment:
Did you have a particular interest in adding support for the media comment params for this commit? If not, it seems like it might might make sense to omit these params for now, and we can introduce them later when there is better support around media comment generating/listing in the API.

If we decide to keep them here, it looks like we'll need to do some additional validation:
- verify a MediaComment of type `media_comment_type` with ID `media_comment_id` exists, before setting the media_comment_id on the submission_comment. If it doesn't exist, handle that case gracefully.
- verify the current_user has permissions to associate the media comment with the submission comment.